ecdsa/src/rfc6979.rs

```diff
@@ -87,17 +89,18 @@ where
     }
 
     /// Get the next `HMAC_DRBG` output
-    pub fn next(&mut self) -> GenericArray<u8, D::OutputSize> {
-        self.k.update(&self.v);
-        let t = self.k.finalize_reset().into_bytes();
+    pub fn generate_to_slice(&mut self, out: &mut [u8]) {
```

Comment:
We've typically used a `*_into` naming convention for this sort of API elsewhere in the project. How about:

```suggestion
    pub fn generate_into(&mut self, out: &mut [u8]) {
```

Reply:
There's also `XofReader` for precedent: https://docs.rs/digest/0.9.0/digest/trait.XofReader.html

Reply:
Let's go with `generate_into` then.